### PR TITLE
rpc: implement gettxout RPC

### LINKED ISSUE #345: RPC: implement gettxout

### PR DESCRIPTION
**Pull request by fyrchik**
Closes #345 .

### CODE REVIEW

File: pkg/rpc/server.go [hunk 1]
```diff
@@ -245,6 +245,9 @@ Methods:
 		getrawtransactionCalled.Inc()
 		results, resultsErr = s.getrawtransaction(reqParams)
 
+	case "gettxout":
```
Comment (roman-khimov) [inline pkg/rpc/server.go]: Missing Prometheus counter.

**Comment by roman-khimov** [comment]
Also, documentation should be updated.

File: pkg/rpc/server_test.go [hunk 2]
```diff
@@ -112,6 +112,38 @@ var rpcTestCases = map[string][]rpcTestCase{
 			fail:   true,
 		},
 	},
+	"gettxout": {
```
Comment (roman-khimov) [inline pkg/rpc/server_test.go]: Hmm, and where is the case with some successful txout result?

Comment (fyrchik) [reply pkg/rpc/server_test.go]: It is tested separately below.

**Pull request merged**